src/System/tests/IntegratorTest.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * @file RecursiveLeastSquareTest.cpp
+ * @authors Giulio Romualdi
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <memory>
+#include <iostream>
+
+// Catch2
+#include <catch2/catch.hpp>
+
+#include <BipedalLocomotion/System/ForwardEuler.h>
+#include <BipedalLocomotion/System/LinearTimeInvariantSystem.h>
+
+using namespace BipedalLocomotion::System;
+
+template <typename T, typename U>
+bool areVectorsEqual(const T& vector1, const U& vector2, const double& tollerance = 0)
```

Comment:
Tolerance has a single "l" :grin:

Reply:
Fixed in 1be19bfe1042e0669070bd43a1be4cce022b2fbb